server/info/lists/server_software.md

```diff
@@ -14,740 +14,937 @@ Found - Previously lost but has been found
 Unreleased - Currently unreleased or was never released  
 Private - Made private after being added to this list
 
+### For Contributors:
+Please add software with the syntax
+https://pastebin.com/txMwgK3p
```

Comment:
why not just put the template into a code block? it's not that long

Reply:
That's a good idea actually